charts/incubator/sdtd/values.yaml

```diff
@@ -0,0 +1,62 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: ghcr.io/reitermarkus/7d2d
+  # -- image tag
+  tag: main
+  # -- image pull policy
+  pullPolicy: Always
+
+# -- environment variables. See more environment variables in the [7d2d documentation](https://github.com/reitermarkus/7d2d).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+  # VERSION: alpha20.2
+  # SERVER_NAME: My 7 Days to Die Server
+
+# -- Configures service settings for the chart.
+# @default -- See values.yaml
+service:
+  main:
+    ports:
+      http:
+        enabled: false
+        primary: false
+      serverquery:
+        primary: true
+        enabled: true
+        port: 26900
+      serverlist:
+        enabled: true
+        port: 26900
+        protocol: UDP
+      steam:
+        enabled: true
+        port: 26901
+        protocol: UDP
+
+podSecurityContext:
+  fsGroup: 1000
```

Comment:
We don't enable/set the podSecurityContext in our charts by default, unless we _really_ have to.

```suggestion
# -- Configure the Security Context for the Pod
podSecurityContext: {}
  # fsGroup: 1000  
```